Documentation/configuration/api-restrictions-table.rst

```diff
@@ -0,0 +1,127 @@
+.. <!-- This file was autogenerated via api-flaggen, do not edit manually-->
+
+Cilium Agent API
+================
+
+The following API flags are compatible with the ``cilium-agent`` flag
+``enable-cilium-api-server-access``.
+
+===================== ====================
+Flag Name             Description
+===================== ====================
+DeleteEndpointID      Deletes the endpoint specified by the ID. Deletion is
+                      imminent and atomic, if the deletion request is valid and
+                      the endpoint exists, deletion will occur even if errors are
+                      encountered in the process. If errors have been encountered,
+                      the code 202 will be returned, otherwise 200 on success. All
+                      resources associated with the endpoint will be freed and the
+                      workload represented by the endpoint will be disconnected.It
+                      will no longer be able to initiate or receive communications
+                      of any sort.
```

Comment:
Edits for clarity, use present tense
```suggestion
DeleteEndpointID      Deletes the endpoint specified by the ID. Deletion is
                      imminent and atomic: if the deletion request is valid and
                      the endpoint exists, deletion occurs even if errors are
                      encountered in the process. If errors are encountered,
                      HTTP code 202 is returned, otherwise 200 returns on success. Upon deletion,
                      all resources associated with the endpoint are freed and the
                      workload represented by the endpoint is disconnected. The endpoint
                      is no longer able to initiate or receive communications
                      of any sort.
```